x-pack/test/dataset_quality_api_integration/tests/data_streams/integration_dashboards.spec.ts

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { DatasetQualityApiClientKey } from '../../common/config';
+import { FtrProviderContext } from '../../common/ftr_provider_context';
+
+interface IntegrationPackage {
+  name: string;
+  version: string;
+}
+
+export default function ApiTest({ getService }: FtrProviderContext) {
+  const registry = getService('registry');
+  const supertest = getService('supertest');
+  const datasetQualityApiClient = getService('datasetQualityApiClient');
+
+  const integrationPackages: IntegrationPackage[] = [
+    {
+      // with dashboards
+      name: 'postgresql',
+      version: '1.19.0',
+    },
+    {
+      // without dashboards
+      name: 'apm',
+      version: '8.4.2',
+    },
+  ];
+
+  async function installPackage({ name, version }: IntegrationPackage) {
```

Comment:
wdyt about extracting these methods (`installPackage`, `uninstallPackage` and `callApiAs`) into a helper file so we can use them from other tests?